esi_leap/db/sqlalchemy/models.py

```diff
@@ -112,3 +112,25 @@ class Lease(Base):
         'Lease',
         backref=orm.backref('child_leases', remote_side=uuid),
     )
+
+
+class Event(Base):
+    """Represents an event."""
+
+    __tablename__ = 'events'
+    __table_args__ = (
+        Index('event_type_idx', 'event_type'),
```

Comment:
I think event_time could also be an index since searching by last_event_time might be a common use case.

Reply:
you're definitely right - good catch! I've added it